code/modules/client/preferences_toggles.dm

```diff
@@ -409,6 +409,26 @@
 		prefs.toggles_sound ^= SOUND_ADMIN_ATMOSPHERIC
 		to_chat(src, SPAN_NOTICE("You will [atmospheric_toggle ? "now" : "no longer"] hear atmospheric admin sounds."))
 
+/client/proc/receive_random_tip()
+	var/picked_type = tgui_alert(src, "What kind of tip?", "Tip Type", list("Marine", "Xenomorph", "Meta")) //no memetips for them joker imp
+	var/message
+	var/list/types_to_pick = list(
+		"Marine" = "strings/marinetips.txt",
+		"Xenomorph" = "strings/xenotips.txt",
+		"Meta" = "strings/metatips.txt"
+	)
+	var/list/tip_list = file2list(types_to_pick[picked_type])
+	if(length(types_to_pick[picked_type]))
+		message = pick(tip_list)
```

Comment:
you can use SAFEPICK() for this (but erroring with a CRASH on failure is still good)